benchmark/tls/secure-pair.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const common = require('../common.js');
+const bench = common.createBenchmark(main, {
+  dur: [5],
+  securing: ['SecurePair', 'TLSSocket'],
```

Comment:
Please add `'securing=SecurePair'` to `./test/sequential/test-benchmark-tls.js` in the array with the other options there. Otherwise the test will run twice the duration it has to.

Reply:
@BridgeAR Done in commit 487a32797b.